pmacApp/powerPmacAsynPortSrc/sshDriver.cpp

```diff
@@ -376,7 +403,7 @@ SSHDriverStatus SSHDriver::flush()
     return SSHDriverError;
   }
   // Read out any remaining bytes from t he channel
-  rc = libssh2_channel_read(channel_, buff, 2048);
+  rc = libssh2_channel_read(channel_, write_buffer_, PPMAC_MAX_CHAR_LENGTH);
```

Comment:
Is it correct that PPMAC_MAX_CHAR_LENGTH, which is half the size allocation of write_buffer_, is being used with it?

Reply:
No this is incorrect, will be fixed with a new define that is 2*PPMAC_MAX_CHAR_LENGTH